lib/geordi/gitpt.rb

```diff
@@ -69,29 +69,33 @@ def load_projects
           client.project(project_id)
         rescue TrackerApi::Errors::ClientError
           puts # Start a new line
-          Geordi::Interaction.warn "Could not load project #{project_id}. Skipped."
+          Geordi::Interaction.warn "Could not access project #{project_id}. Skipping."
         end
       end.compact
     end
 
     def applicable_stories
+      if Util.testing?
+        return ENV['GEORDI_TESTING_NO_PT_STORIES'] == 'true' ? [] : [OpenStruct.new(id: 12, name: 'Test Story')]
+      end
+
       projects = load_projects
       projects.collect do |project|
         project.stories(filter: 'state:started,finished,rejected')
       end.flatten
     end
 
     def choose_story
-      if Util.testing?
-        return OpenStruct.new(id: 12, name: 'Test Story')
-      end
-
       loading_message = 'Connecting to Pivotal Tracker ...'
       print(loading_message)
       stories = applicable_stories
       reset_loading_message = "\r#{' ' * (loading_message.length + stories.length)}\r"
 
-      Geordi::Interaction.fail('No stories found') if stories.empty?
+      Geordi::Interaction.fail('No stories to offer.') if stories.empty?
```

Comment:
Remember that you do not need the `Geordi::` prefix when you're inside the `Geordi` namespace (like with `Geordi::Util` below).